dff/config/superset_dashboard/dashboards/DFF_statistics_dashboard_1.yaml

```diff
@@ -0,0 +1,1315 @@
+    - ROW-Ae7v1prsp2
+    type: CHART
+  CHART-wci7CHiza6:
+    children: []
+    id: CHART-wci7CHiza6
+    meta:
+      chartId: 12
+      height: 66
+      sliceName: Transition counts
+      sliceNameOverride: General transitions distribution
+      uuid: 9fcc7cc1-9257-4c0d-b377-b3a60a8bf3df
+      width: 12
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-Gw0Ffh1lG
+    - ROW-UiCWq3ix1
+    type: CHART
+  DASHBOARD_VERSION_KEY: v2
+  GRID_ID:
+    children:
+    - TABS-Xoi5oUBxZI
+    id: GRID_ID
+    parents:
+    - ROOT_ID
+    type: GRID
+  HEADER_ID:
+    id: HEADER_ID
+    meta:
+      text: DFF statistics dashboard
+    type: HEADER
+  MARKDOWN-8Q9BhcEwva:
+    children: []
+    id: MARKDOWN-8Q9BhcEwva
+    meta:
+      code: '## Current topic
+
+
+        Assuming that we extract the current topic as a slot on each dialog turn,
+        we can build a bar chart to see how various dialog topics are distributed.
+        The first of the two charts below aggregates the counts over dialog turns,
+        while the second one displays the distribution over time.'
+      height: 20
+      width: 12
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-q4CdGUDlq
+    - ROW-K0fThyfqJq
+    type: MARKDOWN
+  MARKDOWN-HncT4Y_WmV:
+    children: []
+    id: MARKDOWN-HncT4Y_WmV
+    meta:
+      code: '## Flow visit ratio monitor
+
+
+        Thus chart aggregates the ratio of visits for each flow over the whole period
+        of service uptime. The aggregation period can be set to shorter time spans
+        using the filters on the left.'
+      height: 17
+      width: 12
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-6zE8noCIsx
+    - ROW-klrE6Idjlp
+    type: MARKDOWN
+  MARKDOWN-LyRuvHp-7m:
+    children: []
+    id: MARKDOWN-LyRuvHp-7m
+    meta:
+      code: '## Weighted transition graph
+
+
+        The two weighted transition graphs emphasize the most frequently visited nodes
+        and transitions, effectively demonstrating the most popular routes inside
+        the dialog graph.
+
+
+        This can reveal characteristic features of user behavior given the current
+        dialog structure
+
+        that call for changes.
+
+
+        <br />
+
+        '
+      height: 24
+      width: 12
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-JCU6rANFP
+    - ROW-FMNDn-yXPC
+    type: MARKDOWN
+  MARKDOWN-NXluIKVHl5:
+    children: []
+    id: MARKDOWN-NXluIKVHl5
+    meta:
+      code: '# Annotations
+
+
+        Annotations are values that your conversational service derives from user
+        requests and stores in the `Context` object. They are produced by `PROCESSING`
+        functions and by some supplemental services in the `Pipeline`.
+
+
+        To make them available in the dashboard, you need to define a custom extractor
+        function for them (see `Extractor functions` tutorial for more information).
```

Comment:
Add links whenever you reference tutorials.

Reply:
Updated